config.toml

```diff
@@ -924,12 +924,14 @@
     MinPeersThreshold                                = 0.8  # 80%
     DelayBetweenRequestsInSec                        = 5    # 5sec
     MaxTimeoutInSec                                  = 7200 # 2h
-    DelayBetweenConnectionNotificationsInSec         = 60   # 1min
+    PeerShardTimeBetweenSendsInSec                   = 7200   # 2h
```

Comment:
maybe change the time to less than round time, in next release

Reply:
will be changed in T1.3.47.0. Good catch!